tests/Dockerfile

```diff
@@ -87,7 +87,8 @@ ENV LIBGUESTFS_BACKEND=direct \
     TMPDIR=/tmp \
     TEMPDIR=/tmp \
     TEST_DATA=/build \
-    TEST_PUBLISH=
+    TEST_PUBLISH= \
+    HOME=/home/user
```

Comment:
OOI, how is this related to running the container on a different OpenShift? The change seems fine to me, I just wonder if I'm missing the impact of this.

Reply:
The ```user``` account does not work in Openshift without the ```anyuid``` SCC. In the ```restricted``` SCC the pod is run as a random user. However we have all our credentials in ```/home/user``` and the random user needs to be able to read them. So we use this  trick to point to the location of the credentials.